pkgs/applications/radio/yaac/default.nix

```diff
@@ -0,0 +1,46 @@
+{ fetchzip
+, jre8
+, lib
+, makeWrapper
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  version = "1.0-beta181";
+  pname = "yaac";
+
+  src = fetchzip {
+    url = "https://www.ka2ddo.org/ka2ddo/YAAC.zip";
```

Comment:
DO we have a versioned download available? otherwise please upload it to web.archive.org

Reply:
I had overlooked that this is not versioned. Thanks.

There is the source code managed by svn in sourceforge.
https://sourceforge.net/p/yetanotheraprsc/code/HEAD/tree/

How about using this?

Reply:
Doing an ant build now with fetchsvn, it's a little funky with the way it's unpacked now though. Reinstating unzip in install phase to unpack the zip the ant build creates. Appears to be working, just need to test the application to ensure it works as expected for usage.

Reply:
Rebuilt locally on my radio station, works as expected built from source. However the build takes a long time, and I'm pretty sure there's something to optimize here now. Will take another look tomorrow.